redirects/7.1/concepts/auth-ldap-ADtips.html

```diff
@@ -0,0 +1 @@
+/content-services/7.1/admin/troubleshoot/
```

Comment:
troubleshoot isn't a folder, it is a markdown page

Reply:
Correct .. see my later comment: https://docs.alfresco.com/content-services/7.1/admin/troubleshoot/